MovieApplication/MovieApplication/Controller/DescriptionViewController.swift

```diff
@@ -21,43 +37,72 @@ class DescriptionViewController: UIViewController {
     // MARK: - Private Methods
 
     private func setupUI() {
-        createUIDescription()
-        createImageView()
-        createDescriptionLabel()
+        view.backgroundColor = .systemBackground
+        createTableView()
+        createRefreshControl()
+        fetchData()
     }
 
-    private func createUIDescription() {}
+    private func fetchData() {
+        NetworkManager.fetchGenericData(url: url) { (movies: Movies) in
```

Comment:
```suggestion
        NetworkManager.fetchGenericData(url: url) { [weak self] movies in
```
Джан ну как так где weak